src/SAWScript/X86.hs

```diff
@@ -405,8 +406,8 @@ callHandler callMap sym = Macaw.LookupFunctionHandle $ \st mem regs -> do
 -- Returns the shared context and the goals (from the Sym)
 -- and the integer is the (aboslute) address of the function.
 translate ::
-  Options -> RelevantElf -> Fun -> IO (SharedContext, Integer, [Goal])
-translate opts elf fun =
+  Options -> FilePath -> RelevantElf -> Fun -> IO (SharedContext, Integer, [Goal])
```

Comment:
Isn't passing the `FilePath` here redundant, given that it's also contained in the `fileName` field of the `Options` argument?

Reply:
Hmm, so it is. I think I thought that was an Options.hs Options. (Probably either or both should get renamed... someday)

Fixed in 52c9b64827039fecfe5fcb619845d8704b429710.